cluster/src/main/java/com/hartwig/pipeline/alignment/bwa/BwaMemCommand.java

```diff
@@ -15,6 +15,8 @@
                 "-Y",
                 "-t",
                 Bash.allCpus(),
+                "-K",
+                "10000000",
```

Comment:
Could you pull this into a constant just to try to clarify why it's here?

Reply:
Done